### PR TITLE
nvmrc for node20

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1932** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1932?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1932?utm_source=stack-comment).

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18.18.0
```
Comment (karthikbtf) [inline .nvmrc:1]: use 20?

Comment (nickgraz) [reply .nvmrc:1]: Whoops, committed from the wrong terminal!

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**